itest/loadtest/loadtest-sample.conf

```diff
@@ -2,11 +2,20 @@
 network=regtest
 
 # The name of the test case to run. Example "send" or "mint"
-test-case="mint"
+test-case="mintV2"
 
 # Batch size for mint test
 mint-test-batch-size=5
 
+# Total number of groups that assets belong to.
+mint-test-total-groups=20
+
+# Max supply per asset mint.
+mint-test-supply-max=500000
+
+# Min supply per asset mint.
+mint-test-supply-min=100000
```

Comment:
What do we mean by "supply" here? Asset amounts?

Reply:
yes, amount of asset to mint